customer/urls.py

```diff
@@ -3,4 +3,8 @@
 
 urlpatterns = [
     path("signup/", views.CustomerSignupView.as_view(), name="customer_signup"),
+    path(
```

Comment:
here the routes should be in theis format 
[
"customers/", views.ListAllCustomersView.as_view(), name="customer_details"
"customers/<int:id>", views.CustomerDetailsView.as_view(), name="customer_details"
]

2- the ListAllCustomersView should be called CustomersView  to have a more consistent naming

Reply:
if this urls file have the "customers" prefix in the waddi.urls  already make it a  
[
"/", views.ListAllCustomersView.as_view(), name="customer_details"
"/int:id", views.CustomerDetailsView.as_view(), name="customer_details"
]

Reply:
yes kont lesa ha'ol